test-ciphers.sh

```diff
@@ -1,56 +1,70 @@
-#!/usr/bin/env bash
+#!/bin/bash
```

Comment:
This is less portable the `#!/usr/bin/env bash` and inconsistent with other console scripts. I'd undo this change.

Reply:
will do.